beacon-chain/rpc/eth/light-client/helpers.go

```diff
@@ -126,25 +127,20 @@ func createLightClientBootstrapCapella(ctx context.Context, state state.BeaconSt
 	if beacon == nil {
 		return nil, fmt.Errorf("could not get beacon block header")
 	}
+
+	executionPayloadHeader, err := block.Body().Execution()
+	if err != nil {
+		return nil, fmt.Errorf("could not get execution payload header: %s", err.Error())
+	}
+	executionPayloadProof, err := blocks.PayloadProof(ctx, block)
+	var executionPayloadProofStr []string
+	for i, proof := range executionPayloadProof {
+		executionPayloadProofStr[i] = hexutil.Encode(proof)
```

Comment:
This will panic, you need to use `make` and create a slice with the correct length